docs/csharp/tour-of-csharp/arrays.md

```diff
@@ -0,0 +1,82 @@
+---
+title: C# Arrays | A tour of the C# language
```

Comment:
I'm wondering if we want to establish a convention of which product names come after the pipe. I'd think we'd want just C# here.